# giflib is using some old & deprecated constants which are not available on android
# this patch define the old with the new values
--- /dev/null               2016-12-16 01:29:29.990016308 +0100
+++ giflib/lib/android.h    2017-01-02 10:59:14.228610199 +0100
@@ -0,0 +1,3 @@
+# define S_IREAD        S_IRUSR
+# define S_IWRITE       S_IWUSR
+# define S_IEXEC        S_IXUSR

